.buildkite/scripts/generate_auditbeat_pipeline.sh

```diff
@@ -0,0 +1,161 @@
+  cat > $pipelineName <<- YAML
+
+steps:
+  - group: "Auditbeat Mandatory Testing"
+    key: "mandatory-tests"    
+
+    steps:
+      - label: ":ubuntu: Unit Tests"
+        command:
+          - ".buildkite/auditbeat/scripts/unit-tests.sh"
+        notify:
+          - github_commit_status:
+              context: "Auditbeat: linux/Unit Tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":rhel: Unit Tests"
+        command:
+          - ".buildkite/auditbeat/scripts/unit-tests.sh"
+        notify:
+          - github_commit_status:
+              context: "Auditbeat: rhel/Unit Tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_RHEL9}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":windows:-{{matrix.image}} Unit Tests"
+        command: ".buildkite/scripts/win_unit_tests.ps1"
+        notify:
+          - github_commit_status:
+              context: "Auditbeat: windows/Unit Tests"
+        agents:
+          provider: "gcp"
+          image: "{{matrix.image}}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 200
+          disk_type: "pd-ssd"
+        matrix:
+          setup:
+            image:
+              - "${IMAGE_WIN_2016}"
+              - "${IMAGE_WIN_2022}"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+
+      - label: ":linux: Crosscompile"
+        command:
+          - ".buildkite/auditbeat/scripts/crosscompile.sh"
+        env:
+          GOX_FLAGS: "-arch amd64"
+        notify:
+          - github_commit_status:
+              context: "Auditbeat: Crosscompile"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+YAML
+else
+  echo "The conditions don't match to requirements for generating pipeline steps."
+  exit 0
+fi
+
+echo "Check and add the Extended Tests into the pipeline"
+
+if are_conditions_met_arm_tests; then
+  cat >> $pipelineName <<- YAML
+  - group: "Extended Tests"
+      key: "extended-tests-arm"
+      steps:
+        - label: ":arm: ARM64 Unit Tests"
+          key: "extended-arm64-unit-tests"
+          command: ".buildkite/scripts/unit_tests.sh"
+          agents:
+            provider: "gcp"
+            image: "${IMAGE_UBUNTU_ARM64}"
+            machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+          artifact_paths: "${BEATS_PROJECT_NAME}/build/*.*"
+YAML
+fi
+
+if are_conditions_met_win_tests; then
+  cat >> $pipelineName <<- YAML
+    - group: "Windows Extended Testing"
+      key: "extended-tests-win"
+      steps:
+        - label: ":windows: Windows 2019 Unit Tests"
```

Comment:
Why is there no GitHub check for the steps in `Group: "Windows Extended Testing"`?